python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -346,12 +346,6 @@ def __init__(self, path: str) -> None:
         path = preprocess_ondisk_dataset(path)
         with open(path) as f:
             self.yaml_data = yaml.load(f, Loader=yaml.loader.SafeLoader)
```

Comment:
How can user override the config before load? should we consider exposing yaml_data as attribute?

    @property
    def yaml_data(self) -> object:
        """Return the yaml data."""
        return self._yaml_data

Also make dataset_dir -> _dataset_dir, yaml_data-> _yaml_data

Reply:
Got